widget/contrib/gpmdp.lua

```diff
@@ -8,79 +8,96 @@
 
 local helpers             = require("lain.helpers")
```

Comment:
All of these will go. `watch` and `focused` will be called from `awful` in `rc.lua`, while for `json`, you can write a note like in bitcoin example [here](https://awesomewm.org/recipes/watch).